Explain the tests being run against this project

danielbachhuber opened the pull request.

pantheon-circleci commented:
@danielbachhuber, thanks for your PR! By analyzing the blame information on this pull request, we identified @joshkoenig, @allan23 and @ataylorme to be potential reviewers

danielbachhuber commented:
@ataylorme #reviewmerge

ataylorme commented on README.md:
```diff
@@ -39,6 +39,17 @@ This plugin is under active development on GitHub:
+
+Both of these test suites can be run locally, with a varying amount of setup.
+
+PHPUnit requires the [WordPress PHPUnit test suite](https://make.wordpress.org/core/handbook/testing/automated-testing/phpunit/), and access to a database with name `wordpress_test`. If you haven't already configured the test suite locally, you can run `bash bin/install-wp-tests.sh wordpress_test root '' localhost`. You'll also access to a running Solr instance, in order to run the unit tests aga ...
```
> You'll also access to a running Solr instance, in order to run the unit tests against Solr.

@danielbachhuber let's update this to "_You'll also need access to a running Solr instance, in order to run the unit tests against Solr._" (adding the word *need*).

danielbachhuber replied:
0b51df0

ataylorme commented on README.md:
```diff
@@ -39,6 +39,17 @@ This plugin is under active development on GitHub:
+
+PHPUnit requires the [WordPress PHPUnit test suite](https://make.wordpress.org/core/handbook/testing/automated-testing/phpunit/), and access to a database with name `wordpress_test`. If you haven't already configured the test suite locally, you can run `bash bin/install-wp-tests.sh wordpress_test root '' localhost`. You'll also access to a running Solr instance, in order to run the unit tests aga ...
+
+Behat requires a Pantheon site with Solr enabled. Once you've created the site, you'll need install Terminus, and set the `TERMINUS_TOKEN`, `TERMINUS_SITE`, and `TERMINUS_ENV` environment variables. Then, you can run `./bin/behat-prepare.sh` to prepare the site for the test suite.
```
> you'll need install Terminus

@danielbachhuber change to "_you'll need to install Terminus_" (adding the word *to*). Let's also link "_install Terminus_" to https://github.com/pantheon-systems/terminus#installation

danielbachhuber replied:
1202d59

ataylorme commented:
@danielbachhuber I added some notes. Please make the same updates to `readme.txt` as well.

danielbachhuber commented:
> I added some notes.

Good finds. Updated.

> Please make the same updates to readme.txt as well.

Oh — did you know you can just edit `readme.txt`, and then use `grunt readme` to compile the README.md?

ataylorme commented:
> Oh — did you know you can just edit `readme.txt`, and then use grunt readme to compile the README.md?

Nope - I'll definitely use that in the future.

> Good finds. Updated.

Awesome, merging this in.
